common/kadai-common/src/main/java/io/kadai/common/internal/util/LogSanitizer.java

```diff
@@ -18,6 +18,7 @@
 
 package io.kadai.common.internal.util;
 
+// TODO FIXME replace with OWASP SANITIZER: https://github.com/OWASP/owasp-java-encoder
```

Comment:
I suggest we rather open an issue than create a TODO comment.

Reply:
Done: https://github.com/kadai-io/kadai/issues/69